sync_diff_inspector/config/config.go

```diff
@@ -155,17 +156,26 @@ func (d *DataSource) ToDBConfig() *dbutil.DBConfig {
 
 // RegisterTLS register TLS config for driver
 func (d *DataSource) RegisterTLS() error {
+	var err error
+	var sec *Security
+	var tlsConfig *tls.Config
+
 	if d.Security == nil {
-		return nil
+		d.Security = &Security{}
+		sec = d.Security
+		tlsConfig = &tls.Config{
+			InsecureSkipVerify: true, // #nosec G402
```

Comment:
dumping can export min TLS version

https://github.com/pingcap/tidb/blob/55cf58bf1681fe1d584e95cede4fa49c7197682a/dumpling/export/config.go#L282-L288

and DM sets it to VersionTLS10

https://github.com/pingcap/tiflow/blob/b949fa6674958aca43dc55c711e0fc41a6600588/dm/dumpling/dumpling.go#L433

I remember the decision is that user may still need to migaret from old MySQL instances. So we should allow old TLS version. It's also friendly for sync-diff to do this.